hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/handlers/ContractCreateHandlerParityTest.java

```diff
@@ -35,6 +35,7 @@
 import com.hedera.test.factories.scenarios.TxnHandlingScenario;
```

Comment:
What does the name of this class mean?  (i.e., what's the `Parity`?)

Reply:
Unknown, I needed to touch because of the gas calculator

Reply:
Sigh.